PSJira/Public/Set-JiraIssue.ps1

```diff
@@ -41,6 +41,10 @@ function Set-JiraIssue
         [Parameter(Mandatory = $false)]
         [String] $Description,
 
+        # New FixVersion of the issue
+        [Parameter(Mandatory = $false)]
+        [System.Collections.Hashtable] $FixVersion,
```

Comment:
The Jira field is name in plural form.
If out parameter were to be names `-FixVersions` as well, powershell would accept `-FixVersion`. But not the other way around.

_This is just a suggestion. Feel free to ignore it_